### ISSUE TITLE
Same noise generated by generate_data.py for both detectors

### ISSUE THREAD

**Issue by vivinousi** [gwastro/ml-mock-data-challenge-1#5]
The noise generated by generated_data.py for dataset 1 seems to be the same for both detectors, as checked by
```
np.allclose(rawdata[0], rawdata[1], rtol=1e-45, atol=1e-36)
```
in [example_torch.py#L93](https://github.com/gwastro/ml-mock-data-challenge-1/blob/3debc9b0101fd4d5baa325493a2cf2309e5b0319/examples/example_torch.py#L93)

**Comment by MarlinSchaefer** [comment]
Thank you for reporting this issue.
This is caused by the seed in noise generation being the same for all detectors.
It mainly affects data generation for data set 1. However it also has an influence on data sets 2 and 3, because the basic noise is the same. Just the coloring between the two is different.

**Comment by MarlinSchaefer** [comment]
Solved by #6
